app/Http/Controllers/V2NewsController.php

```diff
@@ -11,12 +11,17 @@
 
 class V2NewsController extends Controller
```

Comment:
Create V2ShortnewsController instead. Use a V2BlogController as a base and create new index/show/create/store/edit/update methods